components/scripts/lib/build-scan-offline.sh

```diff
@@ -1,18 +1,12 @@
 #!/usr/bin/env bash
 
-readonly GRADLE_ENTERPRISE_LICENSE="${SCRIPT_DIR}/gradle-enterprise.license"
-readonly READ_BUILD_SCAN_DATA_JAR="${LIB_DIR}/build-scan-clients/read-build-scan-data-cmdline-tool-${SCRIPT_VERSION}-all.jar"
+readonly READ_BUILD_SCAN_DATA_JAR="${LIB_DIR}/build-scan-clients/build-scan-summary-${SUMMARY_VERSION}.jar"
```

Comment:
Should we call this `BUILD_SCAN_SUMMARY_JAR` now?

Reply:
Good idea. Done. Also fixed this to actually point to the jar.